app/views/index/privacy.scala.html

```diff
@@ -9,18 +9,25 @@ <h2 class="heading-large">Privacy</h2>
 	    <p>
 	        This service is run by Civil Service Resourcing. We'll handle the information you give to us according to the
 	        <a href="https://www.gov.uk/data-protection/the-data-protection-act" rel="external" target="_blank">Data Protection Act</a>.
+	        This service is run by Civil Service Human Resources (CSHR). We'll handle the information you give to us in accordance with the <a href="https://www.gov.uk/data-protection/the-data-protection-act" rel="external" target="_blank">Data Protection Act</a>. Our <a href="@routes.ApplicationController.terms()">terms and conditions</a> apply.
 	    </p>
 	    <p>When you create an account on the service, the personal information you give us will be used:</p>
 	    <ul>
-	        <li>so you can apply for a Civil Service apprenticeship</li>
+	        <li>so you can apply for a Civil Service Fast Track apprenticeship</li>
 	        <li>so that we can contact you about your application</li>
 	        <li>for administration and research purposes</li>
 	    </ul>
+	    <p>We won't share your personal details for reasons not relating to the apprenticeship.</p>
+		<p>We'll never share your detail with any third party without asking for your permission first.</p>
```

Comment:
Argh, yeah should be details, this was pasted straight from the doc from the content designer, I should've checked it properly.